docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,324 @@
+
+The formal review process this entails will presumably not be a problem for works the site editors are also reviewing. Mechanically, the process can be lightweight, requiring nothing more than a structured metadata record with a link to a published (web-accessible) OSCAL resource. This record is provided and maintained in NIST's metadata repository for [Management of Institutional Data Assets](https://midas.nist.gov/)). 
+
+With NIST access credentials, consult the [Publishing Instructions on the NIST Intranet](https://inet.nist.gov/adlp/open-access-research-oar/publishing-instructions). If your work has undergone a formal review process (external to ours, such as for a conference or journal), it might be a <q>paper</q>. Even if not (that is, if published on the web site or in a repository), depending on its design and purposes, it might constitute <q>software</q> or <q>data</q>.
+
+Additional Resources linked at the foot of this page include a useful checklist for project documentation such as `readme` documentation. Nothing it describes is mandatory for all projects, but any of this guidance might apply in a given instance, and is worth considering for any project or repository, not only those published formally.
+
+Besides an additional web presence, NIST data publishing provides the following advantages:
+
+- A registered DOI (digital object identifier)
+- Listings in data.nist.gov, [GovInfo](https://www.govinfo.gov/) and other scientific and government indexes (beyond search engines)
+- Long-term archiving (beyond Github) of appropriate artifacts or packages
+- Documentation of authorship and contributions
+- Access and citation tracking
+
+See what data.nist.gov shows currently for OSCAL at the [data.nist.gov portal](https://data.nist.gov/sdp/#/search?q=%20OSCAL).
+
+## Recommended style guides
+
+These excellent style guides have been produced over time but remain current and relevant.
+
+- [Red Hat Technical Writing Style Guide Edition 5.1](https://stylepedia.net/style/5.1/) (open source, incorporating guidance from the IBM Style Guide)
+- Google [developer documentation style guide](https://developers.google.com/style)
+- [Apple Style Guide](https://help.apple.com/applestyleguide/)
+- Carnegie Museums [Web Accessibility Guidelines](http://web-accessibility.carnegiemuseums.org/)
+- Microsoft [Writing Style Guide](https://docs.microsoft.com/style-guide/welcome/)
+
+If you know of a style guide we should list here please make an [Issue on Github](https://github.com/usnistgov/OSCAL/issues/new?labels=User+Story%2C+enhancement%2C+Scope:%20Website&template=feature_request.md).
+
+### NIST style guides
+
+Even if submitting work only for the web site, NIST authors will want to be aware of NIST-specific guidance for its formal publications:
+
+- [Guidance for Authorship of Scholarly and Technical Publications](https://inet.nist.gov/adlp/directives/guidance-authorship-scholarly-technical-publications) (requires NIST authentication)
```

Comment:
Like MIDAS, ditto re access to NIST internal resources on the intranet, unless we really deem it valuable.

Reply:
Okay, but if I get slammed for removing the helpful links....!